BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -121,21 +122,15 @@ extension ElementContent {
     /// - parameter measureFunction: How to measure the `ElementContent` in the given `SizeConstraint`.
     public init(
         measurementCachingKey : MeasurementCachingKey? = nil,
-        measureFunction: @escaping (SizeConstraint) -> CGSize
+        measureFunction: @escaping (SizeConstraint, Environment) -> CGSize
```

Comment:
Fwiw, just changing this vs adding a second method might be a bit of a PITA to adopt in SPOS (IIRC last time I did this there were a lot of these calls), but also is only a one time thing.